### PR TITLE
added some mpeg, scte and id3 references

### PR DESCRIPTION
**Pull request by RufaelDev**

### CODE REVIEW

**Comment by tobie** [comment]
Hi there,

Thanks for your contribution. There's a bunch of syntax errors in the JSON file (essentially missing comas), which you'll want to fix.

File: refs/biblio.json [hunk 1]
```diff
@@ -2312,7 +2378,7 @@
         ],
         "href": "https://www.w3.org/2001/tag/doc/web-https",
         "title": "Securing the Web",
-        "status": "TAG Finding",
+        "status": "Finding",
```
Comment (tobie) [inline refs/biblio.json]: Please make sure to rebase your changes, as you're undoing past changes.

File: refs/biblio.json [hunk 2]
```diff
@@ -1135,16 +1144,6 @@
         "title": "Digital Audio Compression(AC-3, Enhanced AC-3) Standard v1.3.1",
         "href": "http://www.etsi.org/deliver/etsi_ts/102300_102399/102366/01.03.01_60/ts_102366v010301p.pdf"
     },
-    "EVERGREEN-WEB": {
```
Comment (tobie) [inline refs/biblio.json]: Here again, you'll need to rebase your PR to avoid removing this reference that's been added in the meantime.

Comment (RufaelDev) [reply refs/biblio.json]: ok thanks, I am working on it,

**Pull request closed without merging**